### ISSUE TITLE
测评结果较低的问题

### ISSUE THREAD

**Issue by liukang1811** [huixiancheng/No-CPGNet#4]
你好，我在google drive中下载了训练好的模型（wce 和 wce_two_stage）并分别在kitti数据集中进行测评，测评代码是工程中的evaluate.py 使用kitti中的valid集（08），但是测试wce 和 wce_two_stage的结果分别为：
Epoch 44; car iou: 0.92425615; bicycle iou: 0.34408772; motorcycle iou: 0.327422; truck iou: 0.51052713; other-vehicle iou: 0.19114032; person iou: 0.5734159; bicyclist iou: 0.84467614; motorcyclist iou: 0.0; road iou: 0.81974715; parking iou: 0.31249848; sidewalk iou: 0.6598457; other-ground iou: 0.006617649; building iou: 0.8843032; fence iou: 0.48770225; vegetation iou: 0.86043215; trunk iou: 0.60325515; terrain iou: 0.6995845; pole iou: 0.59409064; traffic-sign iou: 0.46759114; mean iou: 0.53216815

Epoch 39; car iou: 0.94990486; bicycle iou: 0.5258709; motorcycle iou: 0.7058376; truck iou: 0.89650655; other-vehicle iou: 0.47812566; person iou: 0.64498734; bicyclist iou: 0.9207215; motorcyclist iou: 0.01826144; road iou: 0.890772; parking iou: 0.31654277; sidewalk iou: 0.71569484; other-ground iou: 0.0074295653; building iou: 0.88886327; fence iou: 0.5177564; vegetation iou: 0.86766565; trunk iou: 0.6517532; terrain iou: 0.74467117; pole iou: 0.6320016; traffic-sign iou: 0.4729658; mean iou: 0.62349117

命令分别为：CUDA_VISIBLE_DEVICES=0 python -m torch.distributed.launch --nproc_per_node=1 evaluate.py --config config/wce.py --start_epoch 44 --end_epoch 44

CUDA_VISIBLE_DEVICES=0 python -m torch.distributed.launch --nproc_per_node=1 evaluate.py --config config/wce_two_stage.py --start_epoch 39 --end_epoch 39
不知道这里面出现了什么问题

**Comment by huixiancheng** [comment]
Hi. Thx for point out this. I recheck the log in Google Drive and find that It's still cause by mismatch in downsample stage. 
Check the new commit. It should be fixed.

**Comment by huixiancheng** [comment]
Also, results show that `(1,2), 2, 2` down-sample factor is better than `(1,2), (1,2), (1,2)` or `2, 2, 2`. Just similar to original Repo. :rofl:

**Comment by liukang1811** [comment]
> Hi. Thx for point out this. I recheck the log in Google Drive and find that It's still cause by mismatch in downsample stage. Check the new commit. It should be fixed.


Yes, I suspect there might be an issue in the code somewhere. Thank you for your assistance in fixing it.
